extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/MicrosoftTransferExtension.java

```diff
@@ -36,11 +37,16 @@ public class MicrosoftTransferExtension implements TransferExtension {
   private static final String CONTACTS = "CONTACTS";
   private static final String CALENDAR = "CALENDAR";
   private static final String PHOTOS = "PHOTOS";
+  private static final String MEDIA = "MEDIA";
   private static final String OFFLINE_DATA = "OFFLINE-DATA";
+
+  // TODO(jzacsh,wmorland) don't keep adding here - just have the converters invoked automatically
+  // when Media isn't supported on one or the other side of this equation; this is just a WIP
+  // prototype to show the concept of converters at play.
   private static final ImmutableList<String> SUPPORTED_IMPORT_SERVICES =
-      ImmutableList.of(CALENDAR, CONTACTS, PHOTOS);
+      ImmutableList.of(CALENDAR, CONTACTS, PHOTOS, MEDIA);
```

Comment:
Below we add a MEDIA exporter but not an importer so we probably shouldn't mark MEDIA as supported for import on this PR.

Reply:
Agreed - good catch; I'm punting export-half of the equation to [my `autoconverter-forking-imports` branch](https://github.com/jzacsh/data-transfer-project/tree/autoconverter-forking-imports) (not opening a PR because it'll be a while before I can prioritize finishing this)